shotgun_api3/shotgun.py

```diff
@@ -217,10 +218,27 @@ def __init__(self):
 
         self.py_version = ".".join(str(x) for x in sys.version_info[:2])
 
+        # extract the OpenSSL version if we can. The version is only available in Python 2.7 and
+        # only if we successfully imported ssl
+        ssl_version_str = None
+        ssl_version_num = "unknown"
+        try:
+            ssl_version_str = ssl.OPENSSL_VERSION
+        except AttributeError, NameError:
+            pass
+        if ssl_version_str:
+            try:
+                ssl_version_num = re.search("openssl\s*([\S.]+).*", ssl_version_str, 
+                                            flags=re.IGNORECASE).group(1)
```

Comment:
Any worries about relying on the formatting of the SSL version string here?  Could we use it without parsing so that if there are changes in the future it doesn't break things?